docs/workflows/lambda.md

```diff
@@ -10,7 +10,7 @@ AWS provides great getting started guide for building Lambdas in the [developer
 
 Cumulus currently supports the following environments for Cumulus Message Adapter enabled functions:
 
-* [Node.js 12.18](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
+* [Node.js 14.19.1](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
```

Comment:
Technically we're still supporting 12.18 right?   The CMA requirements aren't being updated as part of this?

Reply:
Hmm, no, I talked myself out of it.   Or at least it should be > 12.18?     🤔

Reply:
🤔  so you're suggesting saying that the CMA supports Node versions > 12.18?

Reply:
What I mean is: 

If we update CMA *only* to v14 and release a new version, and assuming that for whatever reason a v14 CMA release build cannot be imported by a v12 lambda, Core will work with *either* CMA version as functionally we haven't made a change to the CMA or the message, so we still support 12.18 tasks.     

As always, this dependency tree is clear as mud.

Reply:
I think we still need to note that we *support* v14 though, right?    So either both, or ">" or something like that.

Reply:
How about https://github.com/nasa/cumulus/pull/2917/commits/b8bfc02d31530b8aaaf72f6fcd4dfb670273e1b1?

Wanted to make it clear that the max we support (due to AWS constraints) is 14.